src/sentry/interfaces/stacktrace.py

```diff
@@ -235,6 +237,9 @@ def get_api_context(self, is_public=False, pad_addr=None, platform=None):
         if self.data and "symbolicator_status" in self.data:
             data["symbolicatorStatus"] = self.data["symbolicator_status"]
 
+        if self.datapath:
+            data["datapath"] = self.datapath
```

Comment:
Why is datapath written to data here?

Reply:
wrote this in because the UI gets significantly restructured data now. this is obsolete with #27231